airflow/settings.py

```diff
@@ -350,6 +377,17 @@ def import_local_settings():
                 if not k.startswith("__"):
                     globals()[k] = v
 
+        # TODO: Remove once deprecated
+        if "policy" in globals():
```

Comment:
Worth supporting people who use 1.10 and 2.0 with the same file?

```suggestion
        if "policy" in globals() and "task_policy" not in globals():
```

Reply:
Good Point